learn/core_concepts/documents.md

```diff
@@ -124,3 +124,17 @@ Since CSV does not support arrays or nested objects, `cast` cannot be converted
 ::: note
 If you don't specify the data type for an attribute, it will default to `:string`.
 :::
+
+### Auto-batching
+
+Auto-batching combines consecutive document addition requests into a batch to be processed together. This significantly speeds up the indexing process.
```

Comment:
If you do anything about my comment on `quick_start.md`, you should probably also change this line.